### PR TITLE
docs: update route rules docs

### PR DESCRIPTION
**Pull request by Baroshem**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

**Comment by Baroshem** [comment]
@vejja please check if this makes more sense.

**Comment by vejja** [comment]
> @vejja please check if this makes more sense.

Absolutely, it does - small details only
By the way #288 is related (OP is setting `strictTransportSecurity`on the custom security property instead of using the native radix header)

File: docs/content/1.documentation/2.headers/1.csp.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default defineNuxtConfig({
   routeRules: {
     '/custom-route': {
       headers: {
-        'Content-Security-Policy': <OPTIONS>
+        'Content-Security-Policy': 'default-src 'self'; img-src https://*; child-src 'none';'
```
Comment (vejja) [inline docs/content/1.documentation/2.headers/1.csp.md:32]: "default-src 'self'; img-src https:; child-src 'none';"

Comment (Baroshem) [reply docs/content/1.documentation/2.headers/1.csp.md]: Thanks, fixed!

**Pull request merged**